src/state/reducers/index.js

```diff
@@ -15,6 +18,10 @@ export default combineReducers({
     },
     authReducer
   ),
+  preferences: persistReducer(
+    { key: 'preferences', storage: new CookieStorage(Cookies) },
```

Comment:
How's this cookie configured? There's any default expiration time?

Reply:
Cookie configuration is set by default, but can actually configure it, default set the cookie as a session cookie.  You can find more info about it on the [`docs`](https://github.com/js-cookie/js-cookie#cookie-attributes)